lib/postgrex/pgpass.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Postgrex.Pgpass do
+  @moduledoc """
+  Parses .pgpass file to obtain database credentials
+  """
+
+  @pgpass_path Application.get_env(:postgrex, :pgpass, Path.join(System.user_home, '.pgpass'))
```

Comment:
Per the postgres docs, no.

The solution to use the PGPASSFILE from the env would obviate the need for this module attribute.